mp/src/game/client/momentum/ui/leaderboards/ClientTimesDisplay.cpp

```diff
@@ -151,6 +153,15 @@ void CClientTimesDisplay::PerformLayout()
 //-----------------------------------------------------------------------------
 void CClientTimesDisplay::ShowPanel(bool bShow)
 {
+    if (m_bToggledOpen)
+    {
+        if (bShow == false)
+        {
+            m_bToggledOpen = false;
+        }
+        return;
```

Comment:
Should this return be behind an `else`? With this code, if you're trying to close the panel you'd have to hit tab twice to close it?

Reply:
`m_bToggledOpen` is set to true when tab is held and the user right clicks. Then, when they release the tab key, `m_bToggledOpen` is set to false and the function returns without doing anything. The next time they press tab `ShowPanel(true)` does nothing because it's already open, and the release of that tab press finally closes the leaderboard.